libero/src/main/groovy/com/connexta/libero/Config.groovy

```diff
@@ -38,8 +45,8 @@ class Config {
         mavenRepo = project.maven.repo
         quickBuild = project.maven.quickbuild
```

Comment:
I'm looking at those options (force, quickBuild, dryRun and gitPush) and I'm wondering if they really belong inside the config file. Maybe we should just give them safe default options (most likely `false`)? The reason they seem odd here is that I can't see someone creating different config files _only_ because those options are different. You'd just expect users to use options to configure those script behaviors, not different files. Thoughts?